core/input/input.cpp

```diff
@@ -89,6 +89,19 @@ Input::MouseMode Input::get_mouse_mode() const {
 	return get_mouse_mode_func();
 }
 
+void Input::_notification(int p_what) {
+	switch (p_what) {
+		case NOTIFICATION_POSTINITIALIZE: {
+			for (const KeyValue<StringName, InputMap::Action> &E : InputMap::get_singleton()->get_action_map()) {
```

Comment:
You need to check that the map is loaded, it isn't on headless mode when running tests etc., so you need to check that `InputMap::get_singleton()` returns non-null

Reply:
No problem! I had to dig a little before I realised why this could happen myself